include/nbl/builtin/hlsl/shapes/beziers.hlsl

```diff
@@ -13,11 +13,40 @@ namespace hlsl
 {
 namespace shapes
 {
+    // Modified from http://tog.acm.org/resources/GraphicsGems/gems/Roots3And4.c
+    // GH link https://github.com/erich666/GraphicsGems/blob/master/gems/Roots3And4.c
+    // Credits to Doublefresh for hinting there
+    // returns the roots, and number of filled in real values under numRealValues
+    float2 SolveQuadratic(float3 c, out int numRealValues)
```

Comment:
Move this to somewhere more relevant. Than shapes
@devshgraphicsprogramming you got any ideas?

Reply:
if its not 2D, then put it into `curves` instead of shapes

solve Quadratic function, keep it with the Bezier because it has a very specific signature (me myself I would have wanted the complex roots), else it probably belongs somewhere in `math` header/folder